lib/overcommit/hook/pre_commit/es_lint.rb

```diff
@@ -12,19 +12,30 @@ module Overcommit::Hook::PreCommit
   #     enabled: true
   #     command: ['npm', 'run', 'lint', '--', '-f', 'compact']
   #
+  # If ESLint returns some errors, you can bindly pass by
+  # setting `bindly_pass` option to true.
```

Comment:
Honestly, I don't think it can be useful.
But I thought that some people might want to pass ahead if something goes wrong with ESLint (I won't do it).